src/api/transports/http.js

```diff
@@ -6,31 +6,44 @@ import Transport from './base';
 const { fetch } = fetchPonyfill(Promise);
 const debug = newDebug('steem:http');
 
+class RPCError extends Error {
+  constructor(rpcError) {
+    super(rpcError.message);
+    this.name = 'RPCError';
+    this.code = rpcError.code;
+    this.data = rpcError.data;
+  }
+}
+
+export function jsonRpcCall(uri, {method, id, params}) {
```

Comment:
I suggest jsonRpcCall() -> jsonRpc() || rpc()

This one's not a big deal, but the shorter names are dryer and a little more elegant.

Reply:
Noted :) RIP in peace